test/DecideSpec.hs

```diff
@@ -29,9 +29,9 @@ validBoundsDecideConfig = do
             maxNodes < 30 &&
             minAmountOfUniqueAtoms == fromIntegral (length availableAtoms)
 
-  percentageOfChanged <- choose (1, 100)
-  percentTrueEntriesLow' <- choose (1, 90)
-  percentTrueEntriesHigh' <- choose (percentTrueEntriesLow', 99) `suchThat` (/= percentTrueEntriesLow')
+  percentageOfChanged <- chooseInt (1, 100)
+  percentTrueEntriesLow' <- chooseInt (1, 90)
+  percentTrueEntriesHigh' <- chooseInt (percentTrueEntriesLow', 99) `suchThat` (/= percentTrueEntriesLow')
```

Comment:
That's a weird condition. Why not simply start the range at `percentTrueEntriesLow' + 1`?